yolox/exp/base_exp.py

```diff
@@ -21,7 +21,7 @@ def __init__(self):
         self.seed = None
         self.output_dir = "./YOLOX_outputs"
         self.print_interval = 100
-        self.eval_interval = 10
```

Comment:
plz don't change default value here.

Reply:
Changed the values for testing locally.
Pushed it by mistake. I'll change it back

Reply:
@manangoel99 not changed back here?